_posts/2016-12-22-Rust-1.14.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: post
+title: "Announcing Rust 1.14.0"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.14.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.14.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.14.0][notes] on GitHub.
+1230 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1140-2016-12-22
+
+### What's in 1.14.0 stable
+
+One of the biggest features in Rust 1.14 isn't actually in Rust 1.14: the
+[rustup tool has reached a 1.0 release][rustup], and is now the recomended way
+to install Rust from the project directly. Rustup does a bit more than just
+install Rust:
+
+> rustup installs The Rust Programming Language from the official release
+> channels, enabling you to easily switch between stable, beta, and nightly
+> compilers and keep them updated. It makes cross-compiling simpler with binary
+> builds of the standard library for common platforms. And it runs on all
+> platforms Rust supports, including Windows.
+
+[rustup]: https://internals.rust-lang.org/t/beta-testing-rustup-rs/3316/203
+
+We had [a previous post about Rustup][prev] back in May. You can learn more
+about it there, or by checking it out [on
+GitHub](https://github.com/rust-lang-nursery/rustup.rs)
```

Comment:
Lacking a period at the end (possibly due to the inline link also containing one)